core/broadcast.go

```diff
@@ -158,7 +163,8 @@ func (b *echoBroadcast) sendout(h []byte, p packet, bypass bool) {
 	// we register we saw that packet and we broadcast it
 	b.hashes.put(h)
 	proto := &drand.DKGPacket{
-		Dkg: dkgproto,
+		Dkg:     dkgproto,
+		Context: common.NewContext(b.version.ToProto()),
```

Comment:
Wait I'm not sure why this context should be inside a DKG packet.  
Do you plan to put the version and other stuff inside all protobuf packets ? If yes, why ? if not, why ? 

It seems to me that we should only put the version when we need to, and for DKG, the thing we need is to identify for which ID this DKG packet is, so this is what we really need, an ID, but the version is already determined by the ID. 

It's like TLS: you advertise the version first then you negotiate then the whole session happens, but you are not sending again the version in subsequent messages. This is done. Here we only need the ID for multiplexing but I guess that'll come in a subsequent PR. In our case, the ID contains everything, it's a hash of multiple parameters including the version. Building a way to get the version from any node IS definitely useful nonetheless but I hardly see why it should be present in this type of messages, given we don't do anything with it.

Or am i missing something ?